public/assets/ts/admin/editor.ts

```diff
@@ -23,38 +23,21 @@ class Editor {
     this.editor = new window["JSONEditor"](container, this.options);
 
     this.name = this.getFormattedURL(this.sourceDataURL.split("path=")[1]);
-    this.createButtons(container);
+    this.attachButtonHandlers();
 
     (async () => {
       this.editor.set(await this.getFileData());
     })();
   }
 
-  private createButtons(container: HTMLElement) {
-    const openButton = document.createElement("button");
-    openButton.innerHTML = "Open Markdown Editor";
-    openButton.disabled = true;
-    openButton.classList.add("open-editor-button", "disabled");
-    openButton.id = "open-editor-btn";
-    openButton.onclick = this.openRichTextModal;
+  private attachButtonHandlers() {
+    const openButton = document.getElementById("open-editor-btn");
+    const saveButton = document.getElementById("save-editor-btn");
 
-    const openButtonContainer = document.createElement("div");
-    openButtonContainer.classList.add("open-editor-container");
-    openButtonContainer.appendChild(openButton);
-    container.appendChild(openButtonContainer);
-
-    const saveButton = document.createElement("button");
-    saveButton.classList.add("save", "pink-button");
+    openButton.innerHTML = "Open Markdown Editor";
```

Comment:
Would it be possible to move the contents of the button into the pug file?

For the `saveButton`, we could move `Save <span class="editor-name"></span>` into pug and change the value of the span from JS.